control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -2125,6 +2127,25 @@ func (r *HostedControlPlaneReconciler) reconcileHostedClusterConfigOperator(ctx
 		return fmt.Errorf("failed to reconcile config operator deployment: %w", err)
 	}
 
+	podMonitor := manifests.ConfigOperatorPodMonitor(hcp.Namespace)
```

Comment:
@sjenning as a side-effect, this means we now scrape request metrics for request to the hosted clusters KAS, do you think that will cause issues with the metric collection in CI?